packages/server/cli/cheatHandler.ts

```diff
@@ -0,0 +1,130 @@
+import * as readline from 'readline';
+import { mobFactory } from '../src/mobs/mobFactory';
+import globalData from '../data/global.json';
+import { itemGenerator } from '../src/items/itemGenerator';
+import { Coord } from '@rt-potion/common';
+
+const itemTypes: Array<string> = globalData.item_types.map((item) => item.type);
+const mobTypes: Array<string> = globalData.mob_types.map((mob) => mob.type);
+
+export const HELP_PROMPT = `Available commands:
+- spawn mob [type] x:[x-coord] y:[y-coord]
+- spawn item [type] x:[x-coord] y:[y-coord]
+- exit: Quit CLI`;
+
+export let rl: readline.Interface;
+
+// initialize the CLI on the server
+export const initializeCli = () => {
+  rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+    prompt: 'potions-server> '
+  });
+
+  rl.on('line', handleCliCommand);
+
+  console.log('Potions server running...');
+  console.log("Potions Cheat CLI enabled. Type 'help' for commands.");
+  rl.prompt();
+};
+
+// close the CLI
```

Comment:
I would change all these comments to JSDocs as well as to use the `function` keyword instead of lambdas, just to fit with the style of the rest of the codebase.